Makefile

```diff
@@ -339,6 +339,7 @@ integration-functional-only: out/minikube$(IS_EXE) ## Trigger only functioanl te
 
 .PHONY: html_report
 html_report: ## Generate HTML  report out of the last ran integration test logs.
+	@#TODO the json needs to be recorded when the go test is run, for timestamps to work properly
```

Comment:
@afbjorklund is this todo still a todo ? (After this pr)

Reply:
Indeed, you can't run test2json after the test without broken timestamps. But at least the "Elapsed" is back again

Reply:
The complication was that the -json needs to go into all the _other_ targets, the ones that generate the .txt